assets/admin.py

```diff
@@ -0,0 +1,5 @@
+from django.contrib import admin
+from assets.models import Asset
+
+
+admin.site.register(Asset)
```

Comment:
Can we have the admin show the name of the asset rather than just its UUID?

Reply:
done